src/storage/src/sink/healthcheck.rs

```diff
@@ -80,7 +80,9 @@ impl SinkStatus {
 
     fn can_transition(old_status: Option<&SinkStatus>, new_status: &SinkStatus) -> bool {
         match old_status {
-            None => true,
+            // Reduces the likelihood of duplicate entries
+            // from misuse.
+            None => new_status != &SinkStatus::Starting,
```

Comment:
its because the `health_operator` manually sets the status to `Starting`, but doesn't store that information anywhere...i suppose i could refactor this differently

Reply:
>its because the health_operator manually sets the status to Starting

nit: can you add that as a comment here?

Reply:
I instead just made this do the correct thing in the `health_operator` (mark the `last_reported_status` as `Starting`)